model-optimizer/extensions/front/mxnet/take_ext.py

```diff
@@ -0,0 +1,33 @@
+"""
+ Copyright (C) 2017-2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from extensions.ops.gather import AttributedGather
+from mo.front.extractor import FrontExtractorOp
+from mo.front.mxnet.extractors.utils import get_mxnet_layer_attrs
+from mo.graph.graph import Node
+
+
+class TakeExtractor(FrontExtractorOp):
+    op = 'take'
+    enabled = True
+
+    @classmethod
+    def extract(cls, node: Node):
+        attrs = get_mxnet_layer_attrs(node.symbol_dict)
+        AttributedGather.update_node_stat(node, {
+            'axis': attrs.int('axis', 0),
```

Comment:
According to documentation (https://mxnet.apache.org/versions/1.7.0/api/python/docs/api/symbol/op/index.html?highlight=take#mxnet.symbol.op.take) take have mode attribute. Which behavior will be with different value of mode attribute ? For clip value and wrap value ? If AttributedGather not supported that cases we need to add exception at least.

Reply:
None of the mxnet modes is supported by our implementation. So we cannot raise exception here because we have to raise the exception in all cases :)